android/src/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -320,6 +321,14 @@ protected void onFinishInflate()
   {
     super.onFinishInflate();
     mPreview = findViewById(R.id.pp__preview);
+    mPreview.addOnLayoutChangeListener((v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom) -> {
+      final int oldHeight = oldBottom - oldTop;
+      final int newHeight = bottom - top;
+      if (oldHeight != newHeight)
+      {
+        mOnPlacePageContentChangeListener.OnPlacePageContentChange();
+      }
```

Comment:
```suggestion
        mOnPlacePageContentChangeListener.OnPlacePageContentChange();
```
We don't use bracers for one-liners.